src/PayloadInterface.php

```diff
@@ -6,28 +6,88 @@
 {
     /**
      * @var integer All good.
+     * @deprecated 1.1.0
+     * @see \Equip\Adr\PayloadInterface::STATUS_OK
      */
     const OK = 10;
 
     /**
      * @var integer Error during domain processing.
+     * @deprecated 1.1.0
      */
     const ERROR = 20;
 
     /**
      * @var integer Invalid domain input.
+     * @deprecated 1.1.0
      */
     const INVALID = 30;
 
     /**
      * @var integer Unknown domain state.
+     * @deprecated 1.1.0
      */
     const UNKNOWN = 40;
 
     /**
+     * @var string
+     */
+    const STATUS_OK = 'OK';
+
+    /**
+     * @var string
+     */
+    const STATUS_CREATED = 'Created';
+
+    /**
+     * @var string
+     */
+    const STATUS_ACCEPTED = 'Accepted';
+
+    /**
+     * @var string
+     */
+    const STATUS_NO_CONTENT = 'No Content';
+
+    /**
+     * @var string
+     */
+    const STATUS_MOVED_PERMANENTLY = 'Moved Permanently';
+
+    /**
+     * @var string
+     */
+    const STATUS_FOUND = 'Found';
+
+    /**
+     * @var string
+     */
+    const STATUS_NOT_MODIFIED = 'Not Modified';
+
+    /**
+     * @var string
+     */
+    const STATUS_BAD_REQUEST = 'Bad Request';
+
+    /**
+     * @var string
+     */
+    const STATUS_UNAUTHORIZED = 'Unauthorized';
+
+    /**
+     * @var string
+     */
+    const STATUS_FORBIDDEN = 'Forbidden';
+
+    /**
+     * @var string
+     */
+    const STATUS_NOT_FOUND = 'Not Found';
```

Comment:
What about adding all the possible statuses? Eventually we'll need something that hasn't been added and then we're jumping through versioning. If we add them all now we can avoid that.